lib/main.dart

```diff
@@ -35,6 +38,11 @@ class MyApp extends StatelessWidget {
           ),
         ),
         textTheme: TextTheme(
+          headline1: TextStyle(
```

Comment:
Headline 1 corresponds to the biggest heading.
Use headline 6 or subtitle 1 for 18 font size.

Check out flutter.dev -> TextTheme class for more info